plugins/copr.py

```diff
@@ -174,6 +180,21 @@ def run(self):
             raise dnf.exceptions.Error(
                 _('Unknown subcommand {}.').format(subcommand))
 
+    def _list_installed_repositories(self):
+        self._list_repos_from_directory("/etc/yum.repos.d")
+        self._list_repos_from_directory("/etc/dnf.repos.d")
```

Comment:
You should not list from both directories. Only from that one, which is being used. The path should be stored in `self.base.conf.reposdir`.